.scalafmt.conf

```diff
@@ -0,0 +1,11 @@
+version = 3.7.11
+runner.dialect = scala212
+preset = default
+align.preset = more
```

Comment:
I don't really care for this alignment, can we stick to the default here?

Reply:
I have updated the `.scalafmt.conf` and rebased/pushed PR with default alignment